### PR TITLE
Node pool commands

### PR DESCRIPTION
**Pull request by sagikazarmark**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
Add node pool create and delete support.

### CODE REVIEW

File: cmd/docs/banzai_cluster.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Manage clusters
 * [banzai cluster import](banzai_cluster_import.md)	 - Import an existing cluster (EXPERIMENTAL)
 * [banzai cluster list](banzai_cluster_list.md)	 - List clusters
 * [banzai cluster node](banzai_cluster_node.md)	 - Work with cluster nodes
+* [banzai cluster nodepool](banzai_cluster_nodepool.md)	 - Work with cluster nodepools
```
Comment (stoader) [inline cmd/docs/banzai_cluster.md]:
```suggestion
* [banzai cluster nodepool](banzai_cluster_nodepool.md)	 - Manage cluster node pools
```

File: cmd/docs/banzai_cluster_nodepool.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## banzai cluster nodepool
+
+Work with cluster nodepools
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool.md]:
```suggestion
Manage  cluster node pools
```

File: cmd/docs/banzai_cluster_nodepool.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### Synopsis
+
+Work with cluster nodepools
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool.md]:
```suggestion
Manage cluster node pools
```

File: cmd/docs/banzai_cluster_nodepool_create.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      --no-color             never display color output
+      --no-interactive       never ask questions interactively
+      --organization int32   organization id
+  -o, --output string        output format (default|yaml|json) (default "default")
+      --verbose              more verbose output
+```
+
+### SEE ALSO
+
+* [banzai cluster nodepool](banzai_cluster_nodepool.md)	 - Work with cluster nodepools
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool_create.md]:
```suggestion
* [banzai cluster nodepool](banzai_cluster_nodepool.md)	 - Manage cluster node pools
```

File: cmd/docs/banzai_cluster_nodepool.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+### Options
+
+```
+      --cluster int32         ID of cluster to nodepool
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool.md]:
```suggestion
      --cluster int32         ID of the cluster of which node pools to manage
```

File: cmd/docs/banzai_cluster_nodepool.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      --cluster-name string   Name of cluster to nodepool
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool.md]:
```suggestion
      --cluster-name string   Name of the cluster of which node pools to manage
```

File: cmd/docs/banzai_cluster_nodepool_create.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+Create a node pool for a given cluster
+
+```
+banzai cluster nodepool create [flags]
+```
+
+### Options
+
+```
+      --cluster int32         ID of cluster to create
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool_create.md]:
```suggestion
      --cluster int32         ID of the cluster to create and add a node pool to
```

File: cmd/docs/banzai_cluster_nodepool_create.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+      --cluster-name string   Name of cluster to create
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool_create.md]:
```suggestion
      --cluster-name string   Name of the cluster to create and add a node pool to
```

File: cmd/docs/banzai_cluster_nodepool_delete.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+Delete a node pool for a given cluster
+
+```
+banzai cluster nodepool delete [NODE_POOL_NAME] [flags]
+```
+
+### Options
+
+```
+      --cluster int32           ID of cluster to delete
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool_delete.md]:
```suggestion
      --cluster int32           ID of the cluster to delete the node pool from
```

File: cmd/docs/banzai_cluster_nodepool_delete.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+      --cluster-name string     Name of cluster to delete
```
Comment (stoader) [inline cmd/docs/banzai_cluster_nodepool_delete.md]:
```suggestion
      --cluster-name string     Name of the cluster to delete the node pool from
```

File: cmd/docs/banzai_cluster.md (hunk 1 above)
Comment (sagikazarmark) [reply cmd/docs/banzai_cluster.md]: This is generated code.

**Pull request merged**